packit_service/worker/handlers/copr.py

```diff
@@ -119,14 +117,7 @@ def get_packit_github_installation_time(self) -> Optional[datetime]:
         return None
 
     def run(self) -> TaskResults:
-        installed_at = self.get_packit_github_installation_time()
```

Comment:
can we remove this method and the DATE_OF_DEFAULT_SRPM_BUILD_IN_COPR constant as well?